### ISSUE TITLE
Create and Edit interfaces use aria attributes.

### ISSUE THREAD

**Issue by brucestull** [brucestull/technology-and-perceptibility#98]
ARIA required and invalid attributes are important for AT users.

Resources:
- [aria-required](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-required)
    - `required` is probably adequate since it is available to about 97% of users worldwide.
- [aria-invalid](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid)

[aria-invalid - Description](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid#description)
The aria-invalid attribute is used to indicate that the value entered into an input field is not in a format or a value the application will accept. This may include formats such as email addresses or telephone numbers. aria-invalid can also be used to indicate that a required field is empty.

The aria-invalid attribute can be used with any typical HTML form element, and is not limited to elements that have an ARIA role assigned.

The attribute should be set with JavaScript as a result of a validation process. If a value is determined to be invalid or out-of-range, set aria-invalid="true" and tell the user there is an error. For better user experience, provide suggestions for how the error can be corrected. Do not set aria-invalid="true" on empty required elements until after the user attempts to submit the form. They may still be working on filling it out.

**Comment by brucestull** [comment]
These are only required if proper HTML semantic elements are not available and are not used.
